nginx-php5/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM unit9/base:latest
+
+MAINTAINER Artiom Vasiliev <artiom@unit9.com>
+
+COPY ["php-fpm.rc", "/etc/service/php-fpm/run"]
+
+RUN apt-get update && \
+    apt-get upgrade -y && \
```

Comment:
`unit9/base` already has up-to-date packages, as it runs `dist-upgrade` during builds. (Theoretically [it has less vulns than base `debian:jessie`](https://hub.docker.com/r/library/debian/tags/jessie/).)

Your image should be an automated build on DockerHub, with builds triggered by updates to base. E.g. see setup for [web-node7](https://hub.docker.com/r/unit9/web-node7/~/settings/automated-builds/)